scripts/fuzz.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
I think you generally disagree but asking anyway: can you please move this to a `main "$@"` function? Rationale in https://github.com/thesofproject/sof-test/issues/740

Reply:
I do, but this is your playground so no arguments here.

Reply:
@andyross your playground too !